sql/updates/reverse-dev.sql

```diff
@@ -770,3 +770,4 @@ AS 'BEGIN END' LANGUAGE PLPGSQL SET search_path TO pg_catalog,pg_temp;
 
 DROP FUNCTION IF EXISTS _timescaledb_functions.get_orderby_defaults(regclass,text[]);
 DROP FUNCTION IF EXISTS _timescaledb_functions.get_segmentby_defaults(regclass);
+DROP FUNCTION IF EXISTS _timescaledb_functions.ts_replace_now_mock();
```

Comment:
Not sure you need this in the upgrade/downgrade scripts since it will only be used in regression tests for debug builds.

Reply:
We don't need this function in latest/reverse-dev, removed.